tests/.coveragerc

```diff
@@ -1,7 +1,7 @@
 [run]
 branch = True
 concurrency = multiprocessing,thread
-data_file = .coverages/.coverage
+data_file = ${RUNTESTS_DIR-.}/.coverages/.coverage
```

Comment:
From the coverage docs you linked, it seems that we also need to set the `parallel` option to True?

*When using this technique, be sure to set the parallel option to true so that multiple coverage.py runs will each write their data to a distinct file.*

Reply:
Thanks, I didn't spot that the first time round.

Looking closer now, I'm noticing that as things stand we already have the wanted behavior described at the [docs for parallel option](https://coverage.readthedocs.io/en/7.3.1/config.html#run-parallel): producing under `tests/.coverages` files stamped with the computer name and a random number.

And if I remove `concurrency = multiprocessing,thread`, I just get a single file in that dir called ".coverage", so I think that's the option that's making this work as intended.

I see that the `concurrency` option was added in coverage 4.0, so the admonition about parallel=True may not have kept up with the times.

Reply:
Great, thanks for checking. I rebased onto latest `main` and I'll merge once the tests pass. Thanks!